src/wallet/wallet.h

```diff
@@ -786,6 +786,21 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     bool FundTransaction(CMutableTransaction& tx, CAmount& nFeeRet, bool overrideEstimatedFeeRate, const CFeeRate& specificFeeRate, int& nChangePosInOut, std::string& strFailReason, bool includeWatching, bool lockUnspents, const std::set<int>& setSubtractFeeFromOutputs, bool keepReserveKey = true, const CTxDestination& destChange = CNoDestination());
 
     /**
+     * Bumps the fee of an opt-in-RBF transaction <txid>,
+     * replacing it with a new transaction <wtxNew>
+     */
+    enum BumpFeeResult
```

Comment:
Enum is not being used in this commit (8993411d7b3e6fe2c4baa917874941ed95c39188 `[Move] move bumpfee core logic to CWallet`), would probably make more sense to define in the next commit.